flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64", "i386"]
```

Comment:
i386 hasn't been supproted for over THREE YEARS.

```suggestion
  "only-arches": ["x86_64"]
```

Reply:
Oh my, somehow I let this slip... My bad : <